src/test/java/com/ncc/aif/ExamplesAndValidationTest.java

```diff
@@ -1223,7 +1223,26 @@ void invalid() {
                 testInvalid("NIST.invalid: edge justification contains at most two mentions");
             }
 
+
             @Test
+            void invalidZeroSpans() {
```

Comment:
This is tested in EdgeJustificationLimit.invalid(). If we want it to be in its own test, we should remove it from EdgeJustificationLimit.invalid().

Reply:
Fixed. Removed zero span test from EdgeJustificationLimit.invalid()